linode/instance/schema_resource.go

```diff
@@ -421,6 +421,23 @@ var resourceSchema = map[string]*schema.Schema{
 		Description: "Whether or not this Instance was created with user-data.",
 		Computed:    true,
 	},
+	"disk_encryption": {
```

Comment:
I intentionally opted to _not_ include `disk_encryption` in the nested disk block since it's inherited from its parent instance.